src/base_classes/active_space_solver.cc

```diff
@@ -284,38 +338,40 @@ make_state_weights_map(std::shared_ptr<ForteOptions> options,
 
 RDMs ActiveSpaceSolver::compute_average_rdms(
```

Comment:
I think I would like for this function to be broken into three parts, with `compute_average_rdms()` calling two specialized functions for the averaged and non-averaged cases.